cropduster/files.py

```diff
@@ -135,6 +135,6 @@ def get_for_size(self, size_slug='original'):
 
         image = Image.get_file_for_size(self, size_slug)
         if size_slug == 'preview':
-            if not os.path.exists(image.path):
+            if not default_storage.exists(image.path):
```

Comment:
I think this needs to be `image.name`

Reply:
Similarly to other comment- `path` ends up being functionally equivalent to name 🤷‍♂

Reply:
(for the record- I was wrong here 😄 - I was testing these changes in an application wherein I created a storage class that overrode the value of `path` to be `name`)